testing/py-webencodings/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=py-webencodings
+_pkgname=webencodings
+pkgver=0.5.1
+pkgrel=0
+pkgdesc="Character encoding for the web"
+url="https://github.com/gsnedders/python-webencodings"
+arch="noarch"
+license="MIT"
+options="!check"
```

Comment:
there's tests as well https://github.com/gsnedders/python-webencodings/blob/master/tox.ini#L1

Reply:
I added the missing tests.